lib/members_page.rb

```diff
@@ -0,0 +1,69 @@
+#!/bin/env ruby
+# encoding: utf-8
+# frozen_string_literal: true
+require 'scraped'
+
+class MembersPage < Scraped::HTML
+  decorator Scraped::Response::Decorator::AbsoluteUrls
+
+  field :member_rows do
```

Comment:
Interestingly, now that I've actually been able to run `rubocop` against this, it isn't complaining about length/complexity _here_ (only for `party_info_for`), presumably because it doesn't realise that `field` is essentially a synonym for `def`. @chrismytton any suggestions for how we can convince rubocop that these should be treated the same as 'normal' methods?